src/connection.h

```diff
@@ -56,6 +56,7 @@ typedef enum {
 
 #define CONN_FLAG_CLOSE_SCHEDULED (1 << 0) /* Closed scheduled by a handler */
 #define CONN_FLAG_WRITE_BARRIER (1 << 1)   /* Write barrier requested */
+#define CONN_FLAG_CLIENT (1 << 2)          /* Connection is of a client - not a cluster link. */
```

Comment:
```suggestion
#define CONN_FLAG_NO_OFFLOAD (1 << 2)          /* Connection should not be offloaded to IO threads. */
```
The goal was to keep the connection layer independent of the the server layer.

Reply:
Changed.